### PR TITLE
Overview with org quick looks

### PR DESCRIPTION
**Pull request by msecret**
Relies on https://github.com/18F/cg-style/pull/195, https://github.com/18F/cg-style/pull/196, https://github.com/18F/cg-style/pull/197

### CODE REVIEW

**Comment by msecret** [comment]
What it looks like so far

screen shot 2016-11-29 at 5 45 08 pm

**Comment by msecret** [comment]
Latest update

screen shot 2016-11-30 at 3 23 22 pm

File: static_src/components/app_count_status.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    this.styler = createStyler(style);
+  }
+
+  worstStatus(apps) {
+    return apps.reduce((previousState, app) => {
+      if (rankWorseState(previousState) < rankWorseState(app.state)) {
+        return app.state;
+      }
+      return previousState;
+    }, 'NONE');
```
Comment (adborden) [inline static_src/components/app_count_status.jsx]: I feel like returning `null` here is more appropriate.

File: static_src/components/app_count_status.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  render() {
+    const props = this.props;
+    let status = 'NONE';
```
Comment (adborden) [inline static_src/components/app_count_status.jsx]: I think `NONE` should be in `appStates`, even if it is an invalid state.

**Comment by adborden** [comment]
Have some suggestions, but looks great :+1:

**Pull request merged**